src/test/git-elegant-configure.bats

```diff
@@ -6,6 +6,14 @@ load fake-read
 setup() {
     fake-pass git "config --global core.commentChar" "|"
     fake-pass git "config --local core.commentChar" "|"
+
+    fake-pass git "config --global user.name" "UserName"
+    fake-pass git "config --global user.email" "UserEmail"
+
+    fake-pass git "config --global user.name UserName"
```

Comment:
This line is a duplicate.

Reply:
@extsoft 
`fake-pass git "config --global user.name" "UserName"` is used to have default value of user.name. (To mock this `_user_name_default=$(git config --global "$_user_name_key")`

`fake-pass git "config --global user.email" "UserEmail"` is used to have default value of user.email. (To mock this `_user_email_default=$(git config --global "$_user_email_key")`

The `fake-pass git "config --global user.name UserName"` and `fake-pass git "config --global user.email UserEmail"` mock the final git command, this: `git config $MODE $(eval "echo -n \$${f}_key") $answer`

So they are not duplicates. It's a mocks for different commands.

Reply:
@AntonKarabaza oh, okay.